Public/Src/Tools/Execution.Analyzer/Analyzers.Core/Xldb/XldbProtobufExtensions.cs

```diff
@@ -679,7 +703,7 @@ public static ProcessPip ToProcessPip(this Process pip, PathTable pathTable, Xld
             var xldbProcessPip = new ProcessPip
             {
                 GraphInfo = parentPip,
-                ProcessOptions = (Options)pip.ProcessOptions,
+                ProcessOptions = pip.ProcessOptions == Process.Options.None ? Options.None : (Options) ((int)pip.ProcessOptions << 1),

```

Comment:
>pip.ProcessOptions == Process.Options.None ? Options.None : (Options) ((int)pip.ProcessOptions << 1), [](start = 33, length = 101)

I don't think this is correct - if it's zero, you are kipping it as zero, otherwise you shift it?

Reply:
Options.None has a value of 1 (I did this since if you shift 0, you will get 0, so I force set it to 1, and everything else can be shifted)